dayinfo.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+
+from errbot import botcmd, BotPlugin
+from datetime import datetime, date
+from calendar import isleap
+
+from pynameday.slovakia import Slovakia as sknameday
+from workalendar.europe import Slovakia as skholiday
```

Comment:
@pepol you are of course right, but I do not know how would I personally go about doing that, other thank doing this in a bit hacky way. I am also not sure if this should be part of the first version -- note that this PR is a direct rewrite of a previous version.